onedal/ensemble/forest.py

```diff
@@ -206,6 +206,9 @@ def _get_onedal_params(self, data):
                 ceil(
                     self.min_samples_split * n_samples)))
 
+        rs_ = check_random_state(self.random_state)
+        seed_ = rs_.randint(0, np.iinfo("i").max)
```

Comment:
What is the point of using underscored variable names in the internal functionality?

Reply:
Good eye, I was following similar conventions that was in daal4py, but its not even a good python convention, the underscore should be before the variable to indicate private.